### ISSUE TITLE
Dim everything except the Tab Organizer popup.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#120]
Originally reported on Google Code with ID 120

```
This extension has got a lot going for it like the sort and search feature--BUT, it
should allow us to Re-GROUP the tabs in the current windows--not just a search window??
We should be able to save the grouping of tabs for current and later use too!  Also,
the popup window should allow all the windows to be lined up in columns, not overlapping.
 That just makes it harder to read them.  Just allow us to adjust the window width
size.  Lastly, on popup, dim the background tab like a lightbox effect to focus on
the tabs. It will be awesome after these fixes!
```

Reported by `gbdesigninc` on 2010-12-30 05:28:31
- **Blocked on**: #11, #91, #121

**Comment by Pauan** [comment]
```
1) BUT, it should allow us to Re-GROUP the tabs in the current windows--not just a search
window??

You can move tabs from one window into another one. You can also move tabs within the
same window. I'm not sure what you're suggesting.


2) We should be able to save the grouping of tabs for current and later use too!

It already saves your list of tabs and windows, so once again I'm not sure what you're
asking for.


3) Also, the popup window should allow all the windows to be lined up in columns, not
overlapping.  That just makes it harder to read them.  Just allow us to adjust the
window width size.

Yeah, I have a feature request filed about that. It'll be done When It's Done(tm).


4) Lastly, on popup, dim the background tab like a lightbox effect to focus on the
tabs.

That's an interesting idea. Do you mean to dim the background of the popup, or to dim
every tab that isn't focused?
```

Reported by `pcxunlimited` on 2011-01-01 08:03:53
- **Status changed**: `HelpNeeded`

**Comment by Pauan** [comment]
```
Version 3.5 adds in a grid-based window layout. Give it a try, and tell me what you
think.
```

Reported by `pcxunlimited` on 2011-01-06 20:24:55
- **Blocked on**: #21

**Comment by Pauan** [comment]
```
About the lightbox effect... are you talking about leaving the popup the same, but making
Chrome darker, while the popup is visible? If so, I can't do that right now, but I
might be able to in the future.
```

Reported by `pcxunlimited` on 2011-01-07 19:13:57

**Comment by Pauan** [comment]
```
Thanks for the reply!  Sorry I didn't try everything out completely.  You are right
about #1.

On #2 though I still don't see how we save the windows before closing Chrome?  Do they
have to all be made a Favorite?  I guess I'm thinking your extension can be more like
a session saver built in the Tab Organizing function so when we restart we can choose
what windows and tabs to reopen? :)

As for #3, I tryed out the grid system but I still can't get all the windows to be
full horizontally--they still overlap.  In other words, I don't want them to overlap.
 I think its cool, but a option to turn this off would be nice or maybe we can just
set the width like in the Grid option?  I'd rather be able to see more of the tab titles
and the windows lined up side by side.

#4 you nailed it..the lightbox effect would separate it from all the other windows
with some contrast and focus on the popup window.  

Lastly, I got a New request for you.  #5 the MOST IMPORTANT one yet! Make a feature
in the Menu drop down to RE-SORT windows by Domain.  This would for instance shuffle
all windows from Yahoo.com (and subdomains) onto a Yahoo window, Bestbuy.com on a Bestbuy
window, etcetera so they wouldn't be scattered all over different windows.  I think
many of us get disorganized with too many windows and at least being able to resort
them makes them more manageable.  I don't know how difficult this would be to do, but
this would be so awesome I and I'm sure others would make a donation for it! :)  Thanks
for your hard work!
```

Reported by `gbdesigninc` on 2011-01-15 13:51:41

**Comment by Pauan** [comment]
```
2) Chrome has this functionality built in. Go to Wrench -> Options -> Basics -> Reopen
the pages that were open last. Then just make sure to exit Chrome with Wrench -> Exit,
and your windows and tabs should be there next time you open Chrome.

There are also session management extensions, if you prefer to go that route. If you
want session management built-into Tab Organizer, please star issue 11. I've given
it plenty of thought, and I haven't figured out a good way to do it yet.


3) The windows shouldn't be overlapping in Grid view. In fact, it's not supposed to
be possible. Could you post a screenshot, please? You need to enable Grid in the options
page. It's in the Windows category.


5) That's been filed as issue 93 for a couple months now. You should star that issue
so you can see when it's implemented. In the meantime, you can use Macros to easily
group tabs by domain. See the FAQ for more information:

http://documentation.tab-organizer.googlecode.com/hg/Tab%20Organizer%20FAQ.html#what-are-macros

Basically, in the Options page, under the Macros category, you'd make one macro for
each domain. So you'd have, say, "require yahoo.com in Yahoo" and "require bestbuy.com
in BestBuy". Then, you can hit Menu -> Macros... -> Apply all macros, which will then
automatically move the tabs for you, into the correct windows.

Be careful, though, since I haven't fixed issue 116 yet. Until that's fixed, it may
cause one or more windows to be closed (I still don't know why).
```

Reported by `pcxunlimited` on 2011-01-16 17:36:31

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2011-01-16 17:37:09
- **Blocked on**: #1, #1

**Comment by Pauan** [comment]
```
Once the version 5 rewrite is done, the only thing missing from your feature requests
is the ability to dim everything except the TO popup. So I'll be renaming this issue.
```

Reported by `pcxunlimited` on 2012-08-27 22:44:54
- **Status changed**: `ExternalDependency`
- **No longer blocked on**: #11, #93, #121

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:04:23
- **Labels added**: Type-External
- **Labels removed**: Type-Feature

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:32:45
- **Labels added**: Type-Visible
- **Labels removed**: Type-External

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:36:10
- **Labels added**: Type-Confirmed
- **Labels removed**: Type-Visible

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:39:18
- **Labels removed**: Priority-Medium

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:47:45
- **Status changed**: `External`
